lib/__tests__/does-not-have-tagname.ts

```diff
@@ -59,14 +59,9 @@ describe('assert.dom(...).doesNotHaveTagName()', () => {
     });
 
     test('fails for missing element', () => {
-      assert.dom(null).doesNotHaveTagName('div');
-
-      expect(assert.results).toEqual([
-        {
-          message: 'Element <unknown> should exist',
-          result: false,
-        },
-      ]);
+      expect(() => assert.dom(null).doesNotHaveTagName('div')).toThrow(
+        'Null target or element was passed'
+      );
```

Comment:
hmm... why change the existing support for `null` that we already have? 🤔

Reply:
`assert.dom(null)` will be throwing the error before `doesNotHaveTagName` gets the chance ^^

Reply:
@Turbo87 restored the valid missing element assertions